lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -399,17 +399,19 @@ def add_exemption(post, options = {})
         post[:payment_method_options][:card][:moto] = true if options[:moto]
       end
 
-      # Stripe Payment Intents does not pass any parameters for cardholder/merchant initiated
-      # it also does not support installments for any country other than Mexico (reason for this is unknown)
-      # The only thing that Stripe PI requires for stored credentials to work currently is the network_transaction_id
-      # network_transaction_id is created when the card is authenticated using the field `setup_for_future_usage` with the value `off_session` see def setup_future_usage below
+      # Stripe Payment Intents now supports specifying on a transaction level basis stored credential information.
```

Comment:
Does this field also enable installments to be supported in Mexico? Wondering if that part of the note should be removed or not.

Reply:
So I'm not really sure what the point of that comment was before, there was never installment logic/flagging nor country specific logic. I don't think it's worth keeping since it's not related to a particular piece of code logic